client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -352,6 +358,10 @@ public void close() throws IOException {
       stop();
    }
 
+   public CounterManager getCounterManager() {
```

Comment:
TBH, I don't know. I miss the API meeting and the minutes talk about a RemoteInfinispan entry point. So I did the simplest thing :P
I'll change to `RemoteCounterManagerFactory` approach :)

Do you want to move the code out from infinispan-hotrod-client? 

I'll also shift the operations code to avoid conflict with the multimap!

Reply:
I don't know if you should do that, especially since we don't have a concept of modularity in Hot Rod protocol, so it seems to make sense here. Also given then discussed discoverability of the API. Regrettably the only takeaway from the discussion seemed to me a vague "we need to play with the API and see..."

Reply:
Added RemoteCounterManagerFactory to keep it symmetric with embedded counter and remote-multimap.
The code was kept in client-hotrod